test/fixtures/libnpmsearch-stream-result.js

```diff
@@ -140,28 +140,6 @@ module.exports = [
       { username: 'isaacs', email: 'i@izs.me' },
     ],
   },
-  {
-    name: 'libnpmhook',
```

Comment:
This is probably ok to remove, but we just want to be 100% sure this wasn't load bearing i.e. covering some line of code in search.  I don't think it was though.